src/__tests__/extensions/replay/sessionrecording.test.ts

```diff
@@ -160,6 +160,7 @@ describe('SessionRecording', () => {
             sessionManager: sessionManager,
             requestRouter: new RequestRouter({ config } as any),
             _addCaptureHook: jest.fn(),
+            consent: { isOptedOut: jest.fn().mockReturnValue(false) },
```

Comment:
```suggestion
            consent: { isOptedOut: () => false },
```

is that equivalent, it's not referenced anywhere else in the file 🤷